tests/testthat/test-options.R

```diff
@@ -34,6 +34,8 @@ test_that("all options used are `options_plumber()` parameters", {
   }
   options_used <- unique(sort(gsub("getOption|\\(|\"|,|'|\\)", "", matches)))
   plumber_options_used <- grep("^plumber", options_used, value = TRUE)
+  deprecated_options <-  c("plumber.swagger.url")
+  plumber_options_used <- plumber_options_used[!plumber_options_used %in% deprecated_options]
```

Comment:
Does order of operations prefer `%in%` to `!`? It feels a bit like you might be coercing the string to a boolean, negating it, and checking for the boolean's presence in `deprecated_options` which would produce a false positive- maybe? I could be wrong here.

Reply:
`%in%` has precedence. Will add parens for clarity